project/ProjectPlugin.scala

```diff
@@ -50,7 +50,7 @@ object ProjectPlugin extends AutoPlugin {
       val scalacheckToolbox: String  = "0.2.5"
       val scalamockScalatest: String = "3.6.0"
       val scalatest: String          = "3.0.6"
-      val skeuomorph: String         = "0.0.5"
+      val skeuomorph: String         = "0.0.6"
```

Comment:
```suggestion
      val skeuomorph: String         = "0.0.7"
```

Reply:
You could apply this change once https://github.com/higherkindness/skeuomorph/pull/77 is merged and released